pkg/subctl/cmd/verify.go

```diff
@@ -179,19 +195,14 @@ func clusterNameFromConfig(kubeConfigPath, kubeContext string) string {
 }
 
 func checkValidateArguments(args []string) error {
-	if len(args) != 2 {
-		return fmt.Errorf("two kubeconfigs must be specified")
+	if len(args) != 2 && len(kubeContexts) != 2 {
```

Comment:
Hmm for clarity perhaps the `kubeContexts` flag should have it's own error, and the old style should have a warning that it's deprecated?

Reply:
I’ve left this test as-is (it’s testing negatives), but there is now a deprecation warning and passing two invalid files produces an error.

Reply:
To be clear, I don’t think it’s worth spending much time on `args`-related subtleties, I want to get rid of them entirely in the next release.

Reply:
I was more concerned with letting users know that args are deprecated :)